aim/web/api/views.py

```diff
@@ -11,29 +11,19 @@
 @general_router.get('/static-files/{path:path}/')
 async def serve_static_files(path):
     from aim import web
-    static_file_name = os.path.join(os.path.dirname(web.__file__), 'ui', 'build', path)
+    static_file_name = os.path.join(os.path.dirname(web.__file__), 'ui_v2', 'build', path)
     compressed_file_name = '{}.gz'.format(static_file_name)
     if os.path.exists(compressed_file_name):
         return FileResponse(compressed_file_name, headers={'Content-Encoding': 'gzip'})
     return FileResponse(static_file_name)
 
 
-@general_router.get('/static/{exp_name}/{commit_hash}/media/images/{path}/')
```

Comment:
@gorarakelyan can you confirm, that this is not needed anymore?

Reply:
@mihran113 yup, let's remove it